src/hw.cpp

```diff
@@ -1,6 +1,20 @@
 #include <iostream>
+#include <entt/entt.hpp>
+#define GLM_ENABLE_EXPERIMENTAL
```

Comment:
what does this do ?

Reply:
This enable `glm::to_string`. Without this, we can't "stringify" vectors